packages/jupyter-ai/src/components/chat-messages/chat-message-delete.tsx

```diff
@@ -15,8 +15,7 @@ type DeleteButtonProps = {
 export function ChatMessageDelete(props: DeleteButtonProps): JSX.Element {
   const request: AiService.ClearRequest = {
     type: 'clear',
-    target: props.message.id,
-    after: false
```

Comment:
Again no issue with this removal. Just want to clarify its original purpose was to support the "clear this and all subsequent messages" features which is required for the "edit message" feature (which is effectively "clear this and all subsequent" + "send edited message").

So you can re-add this or something similar when implementing those features.